src/models/ThemeVariant.ts

```diff
@@ -0,0 +1,4 @@
+export enum ThemeVariant {
+  white,
+  black,
```

Comment:
Minor thing, but no one calls themes 'white' and 'black' ) call it 'dark' and 'light'